### PR TITLE
[docs] Update to use right protocol in redirect URI on iOS for openAuthSessionAsync

### PR DESCRIPTION
**Pull request by drnow4u**
# Why

The correct setup of redirect URI in the authentication server and relationship to other setup is uncleared.

# How

 In PR I add a detailed description.

# Test Plan

Not related.

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [x] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: packages/expo-web-browser/src/WebBrowser.ts [hunk 1]
```diff
@@ -212,6 +212,10 @@ export function dismissBrowser(): void {
  * # On iOS:
  * Opens the url with Safari in a modal using `ASWebAuthenticationSession`. The user will be asked
  * whether to allow the app to authenticate using the given url.
+ * To handle redirection back to the mobile application, the redirect URI set in the authentication server 
+ * has to use the protocol provided as the scheme in `app.json` [expo.scheme](../../config/app/#scheme)
```
Comment (amandeepmittal) [inline packages/expo-web-browser/src/WebBrowser.ts:216]:
```suggestion
 * has to use the protocol provided as the scheme in **app.json** [`expo.scheme`](../../config/app/#scheme) 
```

As per our writing style guide, [filenames have to be bold](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text).

**Review by amandeepmittal — CHANGES_REQUESTED** [review]
Thanks for the updates! I've left one suggestion.

**Comment by MarcinNowak-codes** [comment]
Suggestion applied. Thank you @amandeepmittal

**Review by amandeepmittal — APPROVED** [review]

**Comment by amandeepmittal** [comment]
@MarcinNowak-codes, you'll have to follow the following steps to further docs related changes for this package's API:
- You’ll have to manually update the `unversioned` docs for a specific `.ts` file you've edited and then regenerate the build content and data JSON file (via Expo Tools (`et`):  `et gdad -p <PACKAGE_NAME>`)
- For more info on how to access Expo tools, please have a look at this: https://github.com/expo/expo/tree/main/tools#development
- The data is generated in `.json` files. Once that file is generated, you will see the changes by running docs locally using `yarn dev` (assuming the changes are made to unversioned)

**Review by amandeepmittal — APPROVED** [review]
Thank you!

**Comment by amandeepmittal** [comment]
@MarcinNowak-codes, looks like the check is failing. I guess, it's my fault that I forgot to share the details on updating TypeDoc code, sorry about that!

When you are updating the definition, you'll have to run the command `yarn build` to start the TypeScript build compilation and the make the change in the `src/` folder. This will update the types in the `build/` directory and then follow the steps to run the `et` command as I mentioned before.

Once again, sorry for the inconvenience.

**Review by amandeepmittal — APPROVED** [review]

**Review by tsapeta — APPROVED** [review]
Cool, thanks and congrats on your first contribution to expo! 🎉

**Pull request merged**